Sourcery/Parsing/Composer.swift

```diff
@@ -129,6 +143,9 @@ struct Composer {
 
     private func resolveVariableTypes(_ variable: Variable, of type: Type, resolve: TypeResolver) {
         variable.type = resolve(variable.typeName, type)
+        if let definedInTypeName = variable.definedInTypeName {
+            _ = resolve(definedInTypeName, type)
```

Comment:
I think it will be helpful for future to leave the comment explaining why the result is not used here.